scripts/setup.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+# This assumes you have general prerequisites installed as by:
+# https://github.com/artsy/potential/blob/master/scripts/setup
+
+# Exit if any subcommand fails
+set -e
+
+GREEN='\033[0;32m'
+NO_COLOR='\033[0m'
```

Comment:
No need to address this now, it may be worth using `tput` for color manipulation as it degrades nicely and won't output the color codes to unsupported terminals.

Reply:
Switched to `tput`. Thank you for bringing this up.